scripts/add-plan-to-customer-with-price-adjustments/generate-usage-data/README.md

```diff
@@ -0,0 +1,56 @@
+# Overview
+
+Python3 script which shows how to generate usage data for two customers.
```

Comment:
can you add this to its own folder, maybe:
scripts/usage/

then we can have "generate-usage" and a get-usage script in the future

Reply:
done